src/modules/expression-evaluator/src/evaluateExpression.ts

```diff
@@ -74,18 +77,44 @@ export default async function evaluateExpression(
           const objectIndex = childrenResolved[0].objectIndex ? childrenResolved[0].objectIndex : 0
           const inputObject = params && params.objects ? params.objects[objectIndex] : {}
           const property = childrenResolved[0].property
-          return extractNode(inputObject, property)
+          return extractProperty(inputObject, property)
+        } catch {
+          throw new Error("Can't resolve object")
+        }
+
+      case 'API':
+        let url, urlWithQuery, queryFields, queryValues: string[], returnProperty
+        try {
+          ;[url, queryFields, queryValues, returnProperty] = assignChildNodesToQuery(
```

Comment:
`; ` Maybe a mistake?

Reply:
Nah, prettier puts these in sometimes. I don't fully understand why (not in all cases anyway) but there's some discussion [over here](https://github.com/prettier/prettier/issues/1907) if you're interested. 

(It's a consequence of us not having semi-colons at the ends of lines. Sometimes they are required, and it puts them in at the start of the next line. Not quite sure why it would be required in this case though 🤷‍♂️)